src/app/lib/config/services/persian.js

```diff
@@ -16,6 +16,9 @@ export const service = {
       hasAds: false,
       advertisementLabel: 'آگهی',
     },
+    idxPage: {
```

Comment:
This is worth having a chat about - we have config for radio schedules, and then inside that object we say whether there should be radio schedules at all, or on the front page, (or now on idx pages, on demand radio pages etc... if we follow that pattern). Would be worth following the same pattern (not necessarily the current one!) for all features that can be present or not present on a page based on config!

Reply:
Sure, we should discuss this.

We can follow the approach used for `radioSchedules` here https://github.com/bbc/simorgh/pull/6726/files#diff-4db735a7bcca5796b3f7f8aa71163a47